httpx/client.py

```diff
@@ -580,8 +584,10 @@ def redirect_stream(
                 await response.aclose()
                 raise exc from None
             else:
+                response.history = list(history)
+                await response.aread()
```

Comment:
Yup, good call.

It's possible that at a later date we *might* want to ensure that streaming responses *don't* load an intermediate response bodies (in redirect or auth), but that's only a *"might"*, and would be a seperate issue if we *did* choose to tackle it.